src/ja/server/database/sql/database.py

```diff
@@ -20,143 +20,142 @@ class SQLDatabase(ServerDatabase):
     """
     _metadata: MetaData = None
 
-    def __init__(self, host: str = None, user: str = None, password: str = None):
+    def __init__(self, host: str = None, user: str = None, password: str = None, database_name: str = "jobadder"):
         """!
         Create the SQLDatabase object and connect to the given database.
 
         @param host The host of the database to connect to.
         @param user The username to use for the connection.
         @param password The password to use for the connection.
+        @param database_name The name of the database to use.
         """
         self.scheduler_callback: Callable[["ServerDatabase"], None] = lambda *args: None
         self.status_callback: Callable[["Job"], None] = lambda *args: None
         self.in_scheduler_callback: bool = False
-        if SQLDatabase._metadata:
```

Comment:
Because otherwise you could only create one instance of SQLDatabase per process.
Below the initialization of SQLDatabase._metadata there are object properties being defined.

Reply:
Oh sorry, I somehow thought it was working before, that's why I didn't get why you need to change it.

One more question, out of curiosity, when do we need multiple databases per process?